mteb/leaderboard/benchmark_selector.py

```diff
@@ -46,6 +46,7 @@ class MenuEntry:
                         "MIEB(Img)",
                         "VisualDocumentRetrieval",
                         "JinaVDR",
+                        "REAL_MM_RAG"
```

Comment:
Let us not add it to a benchmark yet due to #3222 (this means that we can merge this without caring about the other PR and once both is merge then we can add both)